frontend/app/admin/src/views/organizations/tags/EditOrganizationTagView.vue

```diff
@@ -11,47 +11,110 @@
                 v-model="name"
                 class="input"
                 type="text"
-                placeholder="Naam van deze rol"
+                placeholder="Naam van deze tag"
                 autocomplete=""
             >
         </STInputBox>
 
-        <div v-if="!isNew && deleteHandler" class="container">
+        <div class="container">
+            <hr>
+            <h2>{{ $t('Tags in deze categorie') }}</h2>
```

Comment:
```suggestion
            <h2>{{ $t('Subtags') }}</h2>
```
Zou de term categorie overal weglaten, het gaat nog steeds om een tag. Tag = category